### PR TITLE
add handle custom objects with string representation

### PR DESCRIPTION
**Pull request by belyy2910**

### CODE REVIEW

**Comment by belyy2910** [comment]
At present "expand" method can't handle custom objects with string representation like MongoDB ObjectId. Just correcting condition and adding missed stringify in "getValues" method fixed the issue

File: package.json [hunk 1]
```diff
@@ -27,5 +27,8 @@
   },
   "scripts": {
     "test": "mocha --reporter spec"
+  },
+  "dependencies": {
+    "is-plain-object": "=3.0.1"
```
Comment (jonkoops) [inline package.json:32]: Not a fan of introducing external dependencies for such a small change, can we do this internally instead?

**Review by jonkoops — CHANGES_REQUESTED** [review]
I'd also like to see some test cases added for the specific types of object that would require this kind of behavior.

**Comment by jonkoops** [comment]
Closing this PR due to lack of response from the author. If there is still interest in working on this feel free to open up a new PR, or ping me and I can re-open this one.

**Pull request closed without merging**